framework/wazuh/core/cluster/unix_server/server.py

```diff
@@ -0,0 +1,49 @@
+import uvicorn
+from fastapi import APIRouter, FastAPI
+from threading import Thread
+from typing import Any
+
+from wazuh.core.cluster.unix_server.config import get_config
+from wazuh.core import common
+
+SERVER_UNIX_SOCKET_PATH = common.WAZUH_RUN / 'config-server.sock'
+
+
+def get_log_config(node: str) -> dict[str, Any]:
+    """
+    Generate a logging configuration with a consistent format.
+
+    Parameters
+    ----------
+    node: str
+        Identifier for the node to include in the log messages.
+
+    Returns
+    -------
+    dict[str, Any]
+        A dictionary containing the logging configuration for Uvicorn.
+    """
+    log_config = uvicorn.config.LOGGING_CONFIG
+    log_config["formatters"]["default"]["fmt"] = f"%(asctime)s %(levelname)s: [{node}] [Config Server] %(message)s"
```

Comment:
Use single quotes (same for the rest of the file)

```suggestion
    log_config['formatters']['default']['fmt'] = f'%(asctime)s %(levelname)s: [{node}] [Config Server] %(message)s'
```

Reply:
It was changed in edcb389a839c82846356d2ca9e981f422df1c442.